.gitignore

```diff
@@ -14,19 +13,14 @@ node_modules/
 .idea
 .vscode
 .github_changelog_generator
-!vendor/
-vendor/*
+vendor/
 !vendor/autoload.php
-!vendor/composer
-!vendor/firebase
-!vendor/ircmaxell
-vendor/composer/installed.json
```

Comment:
Ignoring vendor and not ignoring vendor/composer then ignoring vendor/composer/installed.json does not work in git.